src/Periphery/Permit2Proxy.sol

```diff
@@ -100,6 +100,15 @@ contract Permit2Proxy is WithdrawablePeriphery {
             ) {
                 revert(reason);
             }
+        } catch (bytes memory reason) {
+            if (
+                IERC20(tokenAddress).allowance(msg.sender, address(this)) <
+                amount
+            ) {
+                assembly {
+                    revert(add(reason, 32), mload(reason))
```

Comment:
please use LibUtil.revertWith(reason)

Reply:
ok! Good to know we have this functionality already